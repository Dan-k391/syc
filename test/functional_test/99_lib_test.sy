int main() {
    _sysy_l1[_SYSY_N / 2] =  1;
    // "A%d%c"
    _sysy_l1[0] = 0x25642541;
    _sysy_l1[1] = 0x63;
    putf(_sysy_l1, 1, 0x42);
    return _sysy_l1[_SYSY_N / 2];
}