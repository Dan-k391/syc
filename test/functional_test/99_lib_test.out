A1B
1